arlo_server/jurisdictions.py

```diff
@@ -1,35 +1,139 @@
 from flask import jsonify
+from typing import List, Dict, Optional
+from enum import Enum
+from sqlalchemy import func
 
 from arlo_server import app
-from arlo_server.models import Election, Jurisdiction
+from arlo_server.models import (
+    Election,
+    Jurisdiction,
+    SampledBallotDraw,
+    SampledBallot,
+    Batch,
+    Round,
+    AuditBoard,
+)
 from arlo_server.auth import with_election_access, UserType
+from arlo_server.rounds import get_current_round
 from util.process_file import serialize_file, serialize_file_processing
+from util.jsonschema import JSONDict
 
 
-def serialize_jurisdiction(db_jurisdiction: Jurisdiction) -> dict:
+def serialize_jurisdiction(
+    jurisdiction: Jurisdiction, round_status: Optional[JSONDict]
+) -> JSONDict:
     return {
-        "id": db_jurisdiction.id,
-        "name": db_jurisdiction.name,
+        "id": jurisdiction.id,
+        "name": jurisdiction.name,
         "ballotManifest": {
-            "file": serialize_file(db_jurisdiction.manifest_file)
-            if db_jurisdiction.manifest_file
+            "file": serialize_file(jurisdiction.manifest_file)
+            if jurisdiction.manifest_file
             else None,
-            "processing": serialize_file_processing(db_jurisdiction.manifest_file)
-            if db_jurisdiction.manifest_file
+            "processing": serialize_file_processing(jurisdiction.manifest_file)
+            if jurisdiction.manifest_file
             else None,
-            "numBallots": db_jurisdiction.manifest_num_ballots,
-            "numBatches": db_jurisdiction.manifest_num_batches,
+            "numBallots": jurisdiction.manifest_num_ballots,
+            "numBatches": jurisdiction.manifest_num_batches,
         },
+        "currentRoundStatus": round_status,
+    }
+
+
+class JurisdictionStatus(str, Enum):
+    NOT_STARTED = "NOT_STARTED"
+    IN_PROGRESS = "IN_PROGRESS"
+    COMPLETE = "COMPLETE"
```

Comment:
ok, so I think I may not have been sufficiently direct with my feedback that I feel this would be better as `startedAt` and `completedAt` rather than this enum of states.

At the same time, I can live with this if you find it works as is, so all good :)

Reply:
You were clear, I responded here: https://github.com/votingworks/arlo/pull/398#issuecomment-606176112